packages/wrangler/src/__tests__/pages/pages.test.ts

```diff
@@ -24,6 +24,7 @@ describe("pages", () => {
 
 		Commands:
 		  wrangler pages dev [directory] [-- command..]  🧑‍💻 Develop your full-stack Pages application locally
+		  wrangler pages download                        ⚡️ Download settings from your project
```

Comment:
anything speaking against placing this command at the bottom of the Pages cmds list, after `wrangler pages deploy [directory]`? I guess my thinking is that as a user I'd be first interested to know how to create/deploy a project, before I'd want to know how to download the config. But maybe that's just me?

Reply:
I think this list is sorted alphabetically, but I'm not certain

Reply:
judging by
```
wrangler pages project
wrangler pages deployment
wrangler pages deploy [directory]
```
doesn't seem to be. Not sure if there is any convention in how we list these commands tbh

Reply:
Moved